controllers/users.js

```diff
@@ -10,7 +10,7 @@ const UsersController = {
     user.save((err) => {
       if (err) {
         console.log(err);
-        res.status(409).render("users/new", { error: 'User already exists!' });
+        res.status(409).render("users/new", { error: 'User already exists!', name: req.name, surname: req.surname});
```

Comment:
What conditions does this error appear now?

Reply:
Currently it seems to appear if any of the boxes are left empty